internal/web/kubeclient/client.go

```diff
@@ -245,13 +250,22 @@ func (c *Client) filterNamespacesByAccess(ctx context.Context, namespaces []stri
 		return namespaces, true, nil
 	}
 
+	// Look up the plural for ResourceSet from FluxOperatorKinds.
+	var resourceSetPlural string
+	for _, kind := range fluxcdv1.FluxOperatorKinds {
```

Comment:
nit: I see `findFluxKindInfo` in `internal/web/resource.go`. This function could be extracted to the `api` package and used here (and everywhere else in the codebase where we look up a flux kind by name)